tests/Unit/API/REST/V3/Routes/Donors/GetDonorsRouteTest.php

```diff
@@ -506,19 +506,17 @@ public function testGetDonorsShouldRedactAnonymousDonors()
 
         $this->assertEquals(200, $status);
         $this->assertEquals(2, count($data));
-        $this->assertEquals($donor1->id, $data[0]['id']);
-        $this->assertEquals(0, $data[1]['id']);
 
         $anonymousDataRedacted = [
-            //'id', // This property is Checked above...
```

Comment:
When the data is redacted, the ID should be zero, so we also need to rollback this logic.